implants/imix/src/main.rs

```diff
@@ -235,8 +235,14 @@ async fn main_loop(config_path: String, loop_count_max: Option<i32>) -> Result<(
             None
         },
     };
+    
+    let host_id_file = if cfg!(target_os="windows") {
+        "C:\\ProgramData\\system-id"
```

Comment:
Kyle: C drive isn't always there. I will chuckle when this becomes a problem.
Jack: ship it! If it's a problem it's their fault for using Windows.